doc/docs/master/how-tos/use-transactions-to-run-multiple-commands.md

```diff
@@ -199,10 +199,14 @@ transactions will be executed as soon as you close those
 transactions. To resume the initial transaction, you need to run
 `pachctl resume transaction`.
 
-Because nested transactions are not supported, transactions cannot
+Because nested transactions are not supported, transactions typically cannot
```

Comment:
I'd just remove this sentence, since support for transaction nesting is not relevant to conflict resolution (i.e., a system that supports transaction nesting is not more prone to conflict than one that doesn't.)

Reply:
Agreed - I don't understand what this sentence is trying to communicate, and I don't think it adds much.  Other than that, everything else looks good.